src/Command/ResetTemplatesCommand.php

```diff
@@ -85,6 +85,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->resetter->resetAllIndexes($deleteByPattern);
         }
 
-        return 0;
+        return;
```

Comment:
This change is not good practice and will be deprecated from Symfony 4.4: https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.4.md

Reply:
Reverted.